community/plugins/spring-ai-alibaba-plugin-larksuite-starter/src/main/java/com/alibaba/cloud/ai/service/LarkSuiteService.java

```diff
@@ -0,0 +1,58 @@
+package com.alibaba.cloud.ai.service;
+
+import com.alibaba.cloud.ai.properties.LarkSuiteProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.lark.oapi.Client;
+import com.lark.oapi.service.docx.v1.model.CreateDocumentReq;
+import com.lark.oapi.service.docx.v1.model.CreateDocumentReqBody;
+import com.lark.oapi.service.docx.v1.model.CreateDocumentResp;
+import jakarta.annotation.Resource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.ObjectUtils;
+
+import java.util.function.Function;
+
+/**
+ * @author 北极星
+ */
+public class LarkSuiteService implements Function<LarkSuiteService.Request, Object> {
+
+    private static final Logger logger = LoggerFactory.getLogger(LarkSuiteService.class);
+
+    @Resource
+    LarkSuiteProperties larkSuiteProperties;
```

Comment:
Better use constructor autowire mode since most of the other parts of spring ai are like that.